app/Global.java

```diff
@@ -0,0 +1,38 @@
+import java.net.URL;
+
+import play.*;
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.